File: Java/LC2326/src/ListNode.java
public class ListNode {
    int val;
    ListNode next;

    ListNode() {
        this(0, null);
    }

    ListNode(int val) {
        this(val, null);
    }

    ListNode(int val, ListNode next) {
        this.val = val;
        this.next = next;
    }
}

File: Kotlin/LC21/src/main/kotlin/ListNode.kt
class ListNode(var `val`: Int, var next: ListNode?) {
    constructor(value: Int) : this(value, null)

    override fun toString(): String {
        val sb = StringBuilder()

        var tmp: ListNode? = this.next
        sb.append(this.`val`)
        while (tmp != null) {
            sb.append(" -> ").append(tmp.`val`)
            tmp = tmp.next
        }

        return sb.toString()
    }
}